tools/to_toit_source.toit

```diff
@@ -43,13 +49,33 @@ class Cert:
     if comment: print comment
     if sha_fingerprint != null:
       print "SHA256 fingerprint: $sha_fingerprint"
+    if expiry != null:
+      print "Expiry: $expiry"
+    if subject != null:
+      print "Subject: $subject"
+    hash := tls.add_global_root_certificate data
+
     print "*/"
     if is_deprecated:
       print "$name ::= $(name)_"
       print "$(name)_ ::= parse_ $(name)_BYTES_"
     else:
       print "$name ::= parse_ $(name)_BYTES_"
     print ""
+    print "/**"
+    print "Installs the \"$(mixed_case_name)\""
+    print "  root certificate on this process so that it is used for any"
+    print "  TLS connections that do not have explicit root certificates."
+    if sha_fingerprint != null:
+      print "SHA256 fingerprint: $sha_fingerprint"
+    if expiry != null:
+      print "Expiry: $expiry"
+    if subject != null:
+      print "Subject: $subject"
+    print "*/"
+    print "install_$name.to_ascii_lower -> none:"
```

Comment:
As discussed in person: I think it would be nicer if there were objects representing the (unparsed) certificates.